playbooks/roles/airship-deploy-ucp/tasks/main.yml

```diff
@@ -55,6 +57,17 @@
     - install
     - update_airship_osh_site
 
+- name: Create site network manifests
```

Comment:
why are we separating the network manifest copy from the rest?

Reply:
As now network file has jinja template code so need to be copied as template which is done later.